appinfo/info.xml

```diff
@@ -2,17 +2,20 @@
 <info>
 	<id>password_policy</id>
 	<name>Password policy</name>
-	<namespace>Password_Policy</namespace>
+	<summary>Allows admins to configure a password policy</summary>
 	<description>
 		Allow admin to define certain pre-conditions for password, e.g. enforce a minimum length
 	</description>
-	<licence>AGPL</licence>
-	<author>Bjoern Schiessle</author>
 	<version>1.4.0</version>
+	<licence>agpl</licence>
+	<author>Bjoern Schiessle</author>
+	<namespace>Password_Policy</namespace>
```

Comment:
Just for the record, in theory this is not psr compliant :see_no_evil:

Reply:
I know. But lets fix that in a different PR ;)